spec/unit/type/yumrepo_spec.rb

```diff
@@ -326,27 +326,20 @@
         )
       end
 
-      it 'accepts an empty string' do
-        described_class.new(
-          name: 'puppetlabs',
-          proxy: '',
-        )
-      end
-
```

Comment:
This appears to be broken and didn't surface until now. From what I see we don't support specifying an empty string for proxy, only `_none_` which is then munged to an empty string.

Reply:
Seems that it was added as part of: https://tickets.puppetlabs.com/browse/MODULES-10246
in https://github.com/puppetlabs/puppetlabs-yumrepo_core/pull/22

Reply:
Maybe this should be called `allow(Facter).to receive(:value).with(:operatingsystemmajrelease).and_return('8')`, before creating the instance here